### DISCUSSION TITLE
How to connect Navbar toggle to Sidebar in Flowbite React components.

### DISCUSSION THREAD

**Discussion by PlasticLizard** [themesberg/flowbite#176] [Q&A] [answered]
Hello,

I'm having great luck integrating this library, thank you for creating it. I think I'm missing something obvious, but I'm struggling to connect the Navbar.Toggle component to the Sidebar's collapsed state. Navbar.Toggle doesn't seem to export any event handlers, and the context for Navbar toggle is private.

Is there an easy way to toggle the sidebar's collased state when the navbar.toggle is opened or closed?

Thanks!

Nathan

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hey @PlasticLizard,

I think it would be best if you opened an issue here: https://github.com/themesberg/flowbite-react

Cheers!

**Reply by PlasticLizard** [reply]
Will do, thanks!
https://github.com/themesberg/flowbite-react/issues/244

**Comment by lazerflip1** [comment]
I solved it by creating a hidden button with data-drawer-toggle tag and clicking it by javascript from ts, in my case after being triggered by navigation subscription

**Reply by rasel-gannicus** [reply]
Did you try using that toggle bar after re-rendering your components (I'm not talking about reloading) ?? I think your toggle won't work.
